app/assets/javascripts/discourse/app/static/wizard/components/wizard-step.gjs

```diff
@@ -25,6 +25,12 @@ export default class WizardStepComponent extends Component {
     return this.step.id;
   }
 
+  // We don't want to show the step counter for optional steps after
+  // the "Ready" step.
+  get showStepCounter() {
+    return this.args.step.displayIndex < 5;
```

Comment:
Minor one, but if we don't have the ready step index in a variable somewhere, perhaps we can add a constant?

Reply:
Yeah good point, I didn't like these magic numbers either. Will see what I can do